### PR TITLE
Marketplace Add: Q3 Release

### PR DESCRIPTION
**Pull request by hmorris3293**
## Description
This PR adds the fakespec & svgs for our Q3 release for Marketplace

## Type of Change
 * Non breaking change ('update', 'change')
 
## Note to Reviewers
Full release for prod will need to be coordinated with Marketing. ESTIMATED: August 23rd

### CODE REVIEW

**Review by tiffwong — COMMENTED** [review]
Hmm I don't see aaPanel, Chevereto, JetBackup, MagicSpam, Moodle, Nirvashare, Severalnines, or Splunk on Marketplace 🤔

**Comment by HanaXu** [comment]
@tiffwong There needs to be some additional stuff done in LaunchDarkly, will look into this later

**Review by bnussman — COMMENTED** [review]
Loooks good so far, but I can't get the drawer to open for `NirvaShare`

File: packages/manager/src/features/OneClickApps/FakeSpec.ts [hunk 1]
```diff
@@ -589,6 +689,26 @@ export const oneClickApps: OCA[] = [
     ],
     logo_url: 'assets/nextcloud_color.svg',
   },
+  {
+    name: 'Nirvashare',
```
Comment (bnussman) [inline packages/manager/src/features/OneClickApps/FakeSpec.ts:693]: Capitalize the `s` to fix the drawer not opening for this one. It should like like:

```ts
name: 'NirvaShare',
```

**Review by bnussman — CHANGES_REQUESTED** [review]
Once we get `Nirvashare` fixed this should be good to go (along with updating the feature flags). 

On a another note and out of the scope of this PR we really should fix the drawer animation on first open 
https://user-images.githubusercontent.com/6440455/127390376-055ec08c-2902-4f93-8461-d7215b8cea73.mov

File: packages/manager/src/features/OneClickApps/FakeSpec.ts [hunk 2]
```diff
@@ -15,6 +15,26 @@ export interface Doc {
...
 
 export const oneClickApps: OCA[] = [
+  {
+    name: 'aaPanel',
+    description: `Feature-rich alternative control panel for users who need critical control panel functionality but don’t need to pay for more niche premium features. aaPanel is open source and consistently maintained with weekly updates.`,
+    summary:
+      'Popular open source free control panel with robust features and a mobile app.',
+    related_guides: [
+      {
+        title: 'Deploy Aapanel With Marketplace Apps',
```
Comment (HanaXu) [inline packages/manager/src/features/OneClickApps/FakeSpec.ts:25]: aaPanel

**Review by HanaXu — CHANGES_REQUESTED** [review]
The colored icons for the newly added apps are not visible in the drawer, though they are present in the DOM. Looks like some kind of styling issue 🤔 

image

File: packages/manager/src/features/OneClickApps/FakeSpec.ts [hunk 3]
```diff
@@ -15,6 +15,26 @@ export interface Doc {
...
+  {
+    name: 'aaPanel',
+    description: `Feature-rich alternative control panel for users who need critical control panel functionality but don’t need to pay for more niche premium features. aaPanel is open source and consistently maintained with weekly updates.`,
+    summary:
+      'Popular open source free control panel with robust features and a mobile app.',
+    related_guides: [
+      {
+        title: 'Deploy aaPanel With Marketplace Apps',
+        href:
+          'https://www.linode.com/docs/guides/deploy-aaPanel-with-marketplace-apps',
```
Comment (HanaXu) [inline packages/manager/src/features/OneClickApps/FakeSpec.ts:27]: The capital `P` in this href link might be an issue. We don't capitalize the other apps in the `docs/guides/deploy` links

**Review by tiffwong — APPROVED** [review]
Looks great! The documentation link for several of the apps don't work yet but I'm assuming those will be published closer to when this will be released.

**Review by bnussman — APPROVED** [review]
Looks good now, like @tiffwong said, lets check the links to the Linode guides before this is shipped

**Comment by hmorris3293** [comment]
Doc URLs are updated, you can refer to the "slug:" in the doc PR's that correspond to the release: https://github.com/linode/docs/pulls?q=is%3Apr+is%3Aopen+Marketplace+add%3A

**Review by HanaXu — APPROVED** [review]

**Pull request merged**